### DISCUSSION TITLE
Product meta description for SEO

### DISCUSSION THREAD

**Discussion by Charlio99** [medusajs/medusa#3127] [Feature Requests]
### Problem ###
Right now if you have a product description over 160 characters, the indexers "don't really" parse that description, therefore, that product wouldn't be SEO friendly, and it would probably end up at the bottom results.

### Solution ###
- Add a new field to `Product`, for example `seo_description`.
- Fetch that field from the storefront, so it can be added to the `<head>` `<meta>` tags
- Able to edit on the admin frontend.

That way instead of inserting the 1000 characters product description, we can have a brief SEO friendly description in the meta tags.

**Comment by SalahAdDin** [comment]
But the system itself already tells you how to handle it:
![image](https://github.com/medusajs/medusa/assets/5034215/0bf85a83-271d-42e3-b456-25bdc7c38e13)

**Reply by Charlio99** [reply]
I know, but this is not a good practice, this is limiting the visible description to only 160 chars, without being able to explain further you product, maybe usage recommendations, etc.

Shopify, WooCommerce, etc. have "short description" and "long description". Being "short description" the one used as default for META description for search engines or if you have custom plugins you can have a 3rd description being "meta description" so you can precisely control what the search engines index.

The main thing here: SEO/META description ≠ Visible description for client.

**Comment by Buzut** [comment]
I second this, kinda baffled that it's still not incorporated into the default product module… Limiting the product desc to 160 chars is pretty limited depending on what you're selling.

**Comment by ianrothfuss** [comment]
I agree - coming from Shopify, the Meta Title and Meta Description fields are super important.  We had them automated with the Yoast SEO app based on formatting rules as well.
